### DISCUSSION TITLE
ScriptReference/MonoImporter.GetExecutionOrder

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#670] [Notes]
# ScriptReference/MonoImporter.GetExecutionOrder

https://docs.unity3d.com/ScriptReference/MonoImporter.GetExecutionOrder.html

**Comment by STARasGAMES** [comment]
This function doesn't respect presence of [DefaultExecutionOrderAttribute] on a script. More on that: https://uninomicon.com/defaultexecutionorder

And it is EDITOR-only, so be warned.
